packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -93,6 +94,7 @@ const useStyles = makeStyles({
 
 // this is a stub to move the logic here that should end up here.
 export const ChatList = ({
+  usePremiumApis = true,
```

Comment:
The ticket mentions starting with a default of `false`. Are we deviating from that?